share/spice/expand_url/title_content.handlebars

```diff
@@ -0,0 +1,4 @@
+<h3 class="c-base__title text--primary">
+    <a href='http://{{org_url}}'>{{org_url}}</a>
+</h3>
+<h4 class="c-base__sub">{{context_info}}</h4>
```

Comment:
Actually I forgot about `subtitle`. I tried `description` before changing it to this. `description` was rendering it in the same way.

I'll change it to `subtitle` :+1: